include/roboteam_utils/Circle.h

```diff
@@ -0,0 +1,97 @@
+//
+// Created by emiel on 24-02-20.
+//
+
+#ifndef RTT_CIRCLE_H
+#define RTT_CIRCLE_H
+#include "Rectangle.h"
+#include "Vector2.h"
+
+namespace rtt {
+/**
+ * @brief Represents a circle by storing the center and the radius
+ * @date 24-02-2020
+ * @author Emiel Steerneman
+ */
+class Circle {
+   public:
+    Vector2 center;
+    double radius;
```

Comment:
Any idea how you will handle negative radii?

Reply:
I think taking the absolute value of a radius should be adequate, since two circles with radii of respectively -r and r are equivalent. Also, negative radii might give problems with projections.

Reply:
Negative radii???

Reply:
yes?

Reply:
@RobotJesse someone can now construct a circle with a radius of e.g. -1. I am asking what should happen in that case.

Reply:
I have the solution:
` if (circle.radius <= 0) { computer.download_nasty_virus_to_this_computer(); } `

Reply:
Heathens

Reply:
Does this mean you'll merge my PR?

Reply:
Ignore that, don't merge it yet. I want to do implementation and tests first